src/components/WhatsNext/WhatsNextSection.tsx

```diff
@@ -78,6 +101,12 @@ const WhatsNextSection: React.FunctionComponent<React.PropsWithChildren<WhatsNex
                 Learn more
               </ExternalLink>
             )}
+            <CloseButton
```

Comment:
Should we have a tooltip or modal to notify the user that the card will be hidden permanently(browser scope). I feel UX can be asked for this